src/test/java/com/conveyal/datatools/manager/controllers/api/FeedSourceControllerTest.java

```diff
@@ -91,6 +105,23 @@ public void createFeedSourceWithUrlTest() {
         assertEquals(1, jobCountForFeed(feedSourceWithUrl.id));
     }
 
+    /**
+     * Create some labels, add them to the feed source make them admin only, and check that they don't appear if not an admin
+     */
+    @Test
+    public void createFeedSourceWithLabels() {
+        // Testing the following (important) components of label is as of yet not supported.
```

Comment:
If this method ends up never having any content, then the whole method should be commented out.

Reply:
If we switch to `Labels#feedSourceId` then I will add tests